iocage_lib/ioc_start.py

```diff
@@ -298,6 +299,14 @@ def __start_jail__(self):
 
                 net.append(f'ip6.addr={ip6_addr}')
 
+            if assign_localhost:
+                # Make sure this exists, jail(8) will tear it down if we don't
+                # manually do this.
+                if self.check_aliases('127.0.1.1', '4') != '127.0.1.1':
+                    su.run(['ifconfig', 'lo0', 'alias', '127.0.1.1/32'])
+
+                net.append(f'ip4.addr=127.0.1.1')
```

Comment:
We would be writing `ip4.addr` twice i think, is that desirable ?

Reply:
Yes that's fine, jail will only apply it once